osf/management/commands/data_storage_usage.py

```diff
@@ -426,7 +426,13 @@ def write_summary_data(filename, summary_data, remote_base_folder):
                     auth=bearer_token_auth(DS_METRICS_OSF_TOKEN),
                     stream=True,
             ) as old_file:
-                reader = csv.reader(old_file.iter_lines(), delimiter=',', lineterminator='\n')
+                lines = []
+                for line in old_file.iter_lines():
+                    if isinstance(line, bytes):
```

Comment:
Another case of "we should really be using the helper functions we already have for this," as in https://github.com/CenterForOpenScience/osf.io/pull/9271#discussion_r372575144